addons/refuel/CfgVehicles.hpp

```diff
@@ -1,44 +1,5 @@
 #define MACRO_NOZZLE_ACTIONS \
```

Comment:
Should this not also be removed? I don't think anyone could have included this file from a built mod as it has config in it, it had to be copied, so it's safe to remove.